packages/flutter_tools/test/src/fakes.dart

```diff
@@ -0,0 +1,42 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter_tools/src/device.dart';
+
+/// A fake implementation of the [DeviceLogReader].
+class FakeDeviceLogReader extends DeviceLogReader {
```

Comment:
This is not a mock! renamed 


https://github.com/flutter/flutter/issues/33274